docs/reference/ml/ml-shared.asciidoc

```diff
@@ -603,6 +603,17 @@ If `true`, the output excludes interim results. By default, interim results are
 included.
 end::exclude-interim-results[]
 
+tag::failed-category-count[]
+The number of times that categorization wanted to create a new category but
+couldn't because the job had hit its `model_memory_limit`. This number can
```

Comment:
> This number can be higher... could be anywhere between 1 and failed-category-count.

Is this a really important detail that folks are likely to be confused about?  If not, I think those sentences could be shortened to something like this:

> This count does not track which specific categories failed to be created.

If we need to explain why that's important, perhaps another sentence could be added like this: Therefore you cannot use this value to determine the number of unique categories that were missed(?).